### PR TITLE
Add pull to refresh on mmb list

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/list/FlightListContainer.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import * as React from 'react';
+import {
+  graphql,
+  createRefetchContainer,
+  type RelayRefetchProp,
```
Comment (mrtnzlml) [inline app/MMB/src/list/FlightListContainer.js]: @tbergq Where did you find this type? It is not exported from the Relay package...

Comment (mrtnzlml) [reply app/MMB/src/list/FlightListContainer.js]: _(and therefore it's exported as `any` and you can call whatever you want on the relay property)_

Comment (tbergq) [reply app/MMB/src/list/FlightListContainer.js]:
```
export type RelayRefetchProp = RelayProp & {
    refetch(
        refetchVariables:
            | RelayRuntimeTypes.Variables
            | ((fragmentVariables: RelayRuntimeTypes.Variables) => RelayRuntimeTypes.Variables),
        renderVariables?: RelayRuntimeTypes.Variables,
        callback?: (error?: Error) => void,
        options?: RefetchOptions
    ): RelayRuntimeTypes.Disposable;
};
```
This is in index.ts file from `react-realy`

**Comment by mrtnzlml** [comment]
Integrated, thanks.

**Pull request closed without merging**